webapp/src/gitjson.tsx

```diff
@@ -0,0 +1,693 @@
+                </label>
+                <div className="field">
+                    <div className="ui radio checkbox">
+                        <input type="radio" name="patch" checked={bumpType == "patch"} aria-checked={bumpType == "patch"} onChange={onBumpChange} />
+                        <label>{lf("{0}: patch (bug fixes or other non-user visible changes)", pxt.semver.stringify(vpatch))}</label>
+                    </div>
+                </div>
+                <div className="field">
+                    <div className="ui radio checkbox">
+                        <input type="radio" name="minor" checked={bumpType == "minor"} aria-checked={bumpType == "minor"} onChange={onBumpChange} />
+                        <label>{lf("{0}: minor change (added function or optional parameters)", pxt.semver.stringify(vminor))}</label>
+                    </div>
+                </div>
+                <div className="field">
+                    <div className="ui radio checkbox">
+                        <input type="radio" name="major" checked={bumpType == "major"} aria-checked={bumpType == "major"} onChange={onBumpChange} />
+                        <label>{lf("{0}: major change (renamed functions, deleted parameters or functions)", pxt.semver.stringify(vmajor))}</label>
+                    </div>
+                </div>
+            </div>
+        })
+
+        if (!ok)
+            return
+
+        let newv = vpatch;
+        if (bumpType == "major")
+            newv = vmajor;
+        else if (bumpType == "minor")
+            newv = vminor;
+        const newVer = pxt.semver.stringify(newv)
+        core.showLoading("bumpheader", lf("creating release {0}...", newVer));
+        try {
+            const header = this.parent.state.header;
+            await workspace.bumpAsync(header, newVer)
+            await this.maybeReloadAsync()
+            core.hideLoading("bumpheader");
+            core.infoNotification(lf("GitHub release created."))
+        } catch (e) {
+            this.handleGithubError(e);
+        } finally {
+            core.hideLoading("bumpheader");
+        }
+    }
+
+    private async maybeReloadAsync() {
+        // here, the true state of files is stored in workspace
+        const header = this.parent.state.header;
+        const files = await workspace.getTextAsync(header.id);
+        // save file content from workspace, so they won't get overridden
+        pkg.mainEditorPkg().setFiles(files);
+        // check if we need to reload header
+        const newKey = this.pkgConfigKey(files[pxt.CONFIG_NAME])
+        if (newKey == this.previousCfgKey) {
+            // force render
+            this.parent.setState({});
+            return
+        } else {
+            this.previousCfgKey = newKey
+            await this.parent.reloadHeaderAsync();
+        }
+    }
+
+    private async pullAsync() {
+        core.showLoading("loadingheader", lf("pulling changes..."));
```

Comment:
Capitalize

Reply:
all loading message should be lower cased; just like "loading editor..." or "loading...".